satnav/functions/satnav-writer.js

```diff
@@ -1,17 +1,21 @@
-import {
-  S3Client
-} from '@aws-sdk/client-s3'
+import { S3Client } from '@aws-sdk/client-s3'
+import * as Sentry from '@sentry/serverless'
 
 import parseSqsEvent from '../utils/parse-sqs-event.js'
 
 import { writeSatnavIndex } from '../index.js'
 
+Sentry.AWSLambda.init({
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
```

Comment:
Note that this is gonna send telemetry to sentry from **every** invocation... this opts us in to Sentry perf monitoring where it tracks the response time of every request rather than just errors. The sample rate of 1 means "send to sentry for every request, no filtering"

Reply:
yes, I guess we can tweak or disable when launching or after we understand better our performance profile